QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,91 @@
+
+
+This document is a declaration of software quality for the `libconsole-bridge-dev` ROS external dependency, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `libconsole-bridge-dev` Quality Declaration
+
+The ROS external dependency `libconsole-bridge-dev` **does not claim** to be in the **Quality Level 1** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [External dependencies requirements for Quality Level 1 in REP-2XXX](https://www.ros.org/reps/rep-2004.html).
```

Comment:
May be a different link if requirements will be different for external dependencies (but still point to REP-2004 as desired)